packages/server/package.json

```diff
@@ -56,7 +56,8 @@
     "pg": "^7.18.2",
     "pg-query-stream": "^3.4.2",
     "redis": "^3.0.2",
-    "zxcvbn": "^4.4.2"
+    "zxcvbn": "^4.4.2",
+    "http-proxy-middleware": "^1.0.6"
```

Comment:
This is still a dev only dependency. It should not be part of a prod build.

I've solved this by moving the import into the if env == DEV in the app.js